order.go

```diff
@@ -196,6 +208,28 @@ type Transaction struct {
 	PaymentDetails *PaymentDetails  `json:"payment_details"`
 }
 
+type Fulfillments struct {
+	ID              int        `json:"id"`
+	OrderID         int        `json:"order_id"`
+	Status          string     `json:"status"`
+	CreatedAt       *time.Time `json:"created_at"`
+	Service         string     `json:"service"`
+	UpdatedAt       *time.Time `json:"updated_at"`
+	TrackingCompany string     `json:"tracking_company"`
+	ShipmentStatus  string     `json:"shipment_status"`
+	TrackingNumber  string     `json:"tracking_number"`
+	TrackingNumbers []string   `json:"tracking_numbers"`
+	TrackingUrl     string     `json:"tracking_url"`
+	TrackingUrls    []string   `json:"tracking_urls"`
+	Receipt         Receipt    `json:"receipt"`
+	LineItems       []LineItem `json:"line_items"`
```

Comment:
Just a note: this field and a few others will probably be empty when viewing fulfillments via an `Order`. At least it seems that the the fulfillments on the order are very limited according to [the documentation](https://help.shopify.com/api/reference/order#fulfillments-property). This is just something to keep in mind for now.

Reply:
Cool, i'll keep it in mind when using those items. Thanks!